.github/workflows/test.yml

```
@@ -31,6 +31,28 @@ jobs:
           path: ~/go/bin
           key: ${{ runner.os }}-go-tparse-binary
 
+  build-architectures:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-arch: ["386", "amd64", "arm", "arm64"]
```

I'm not sure we should build for `386`. The architecture may not be supported by Tendermint Core. @marbar3778 @tessr certainly know best

```suggestion
        go-arch: ["386", "amd64", "arm", "arm64"]
```

---

```suggestion
        go-arch: ["amd64", "arm", "arm64"]
```

Tendermint does not support i386

---

I think it should be fine. The problem before was 32-bit architectures, but this was solved. If someone would like they can open an issue on Tendermint to facilitate discussion to adding 386 to co builds. We didn't add it because there wasn't a request for it.